### ISSUE TITLE
interactive_fractal example black screen

### ISSUE THREAD

**Issue by gregjohnson2017** [vulkano-rs/vulkano#1979]
# Template

*   Version of vulkano: Tested on release 0.30.0 and master
*   OS: Windows 10
*   GPU (the selected PhysicalDevice): NVIDIA GeForce GTX 980 Ti (type: DiscreteGpu)
*   GPU Driver: GeForce Game Ready Driver version 516.94

# Issue

When I launch the interactive_fractal example, I just get a black screen. Other examples work for me. 

![image](https://user-images.githubusercontent.com/18707752/190432249-9c15cd4a-e10c-4926-b1c1-7c37f1f2eee4.png)

```
$ cargo run --bin interactive_fractal
warning: unused variable: `handle_type`
   --> vulkano\src\device\mod.rs:612:9
    |
612 |         handle_type: ExternalMemoryHandleType,
    |         ^^^^^^^^^^^ help: if this is intentional, prefix it with an underscore: `_handle_type`
    |
    = note: `#[warn(unused_variables)]` on by default

warning: unused variable: `file`
   --> vulkano\src\device\mod.rs:613:9
    |
613 |         file: File,
    |         ^^^^ help: if this is intentional, prefix it with an underscore: `_file`       

warning: `vulkano` (lib) generated 2 warnings
    Finished dev [unoptimized + debuginfo] target(s) in 0.21s
     Running `.\interactive_fractal.exe`
Usage:
    WASD: Pan view
    Scroll: Zoom in/out
    Space: Toggle between Mandelbrot and Julia
    Enter: Randomize color palette
    Equals/Minus: Increase/Decrease max iterations
    F: Toggle full-screen
    Right mouse: Stop movement in Julia (mouse position determines c)
    Esc: Quit
```

**Comment by kroppt** [comment]
I am using AMD RX 5700 and have issues with the same example, but get slightly different results...

![image](https://user-images.githubusercontent.com/14015672/190433148-77d6390a-4a2c-425a-9e2b-27d2aafdc702.png)

**Comment by GRAYgoose124** [comment]
I've also got a black screen on Arch Linux, using a GTX 1070. No warning to speak of. Other examples also work for me. I'm using the i3 WM, and it seems to affect some programs that expect titles to exist and windows to not resize.

I tried replacing the shader output at [source](https://github.com/vulkano-rs/vulkano/blob/master/examples/src/bin/interactive_fractal/fractal_compute_pipeline.rs#L257). With `vec4(1.0);`, which you'd expect to be a perfectly fine substitution, but instead I get a pretty terrible dump ending with:
```
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: DescriptorSetUpdateError(InvalidBinding { binding: 1 })', examples/src/bin/interactive_fractal/fractal_compute_pipeline.rs:137:10
```
So, I strip out get_color, the palette and the WriteDescriptorSet on L134, and back to black screen. That's as far as I checked before giving up.
